### PR TITLE
Added google map component to Groomer Profile

### PR DESCRIPTION
**Pull request by luisabellan**
* Added google map component to Groomer Profile.
* Installed react-geocode to find latitude and longitude for a given address (to be implemented later on).
* Corrected routes after env variables have been modified in our back-end.
* Made the Groomer profile responsive.

### CODE REVIEW

File: src/components/MyMap/MyMap.js [hunk 1]
```diff
@@ -1,44 +1,27 @@
-import React, { useState, useCallback } from 'react';
-import { GoogleMap, LoadScript } from '@react-google-maps/api';
+import React, { Component } from 'react';
+import { Map, GoogleApiWrapper } from 'google-maps-react';
 
-const containerStyle = {
-  width: '400px',
-  height: '400px',
+const mapStyles = {
```
Comment (williamschwindt) [inline src/components/MyMap/MyMap.js]: Did you mean to use this or can we remove it?

**Review by williamschwindt — CHANGES_REQUESTED** [review]
Looks good! I'm excited to see how it looks in our app! Just a few unused variables we could remove before merging.

File: src/components/pages/CustomerRegistration/CustomerRegistration.js [hunk 2]
```diff
@@ -22,7 +23,7 @@ function RegisterCustomer() {
 
   const onSubmit = data => {
     axios
-      .post('https://labspt12-express-groomer-a-api.herokuapp.com/customers', {
+      .post(`${process.env.REACT_APP_API_URI}customers`, {
```
Comment (coryortega) [inline src/components/pages/CustomerRegistration/CustomerRegistration.js]: there should be a trailing slash at the end of the env variable to keep things consistent

**Review by coryortega — CHANGES_REQUESTED** [review]
looks good, just requesting a change about the trailing slash

File: src/components/pages/CustomerRegistration/CustomerRegistration.js (hunk 2 above)
Comment (luisabellan) [reply src/components/pages/CustomerRegistration/CustomerRegistration.js]: done

File: src/components/MyMap/MyMap.js (hunk 1 above)
Comment (luisabellan) [reply src/components/MyMap/MyMap.js]: deleted

**Review by coryortega — APPROVED** [review]

File: src/index.js [hunk 3]
```diff
@@ -70,6 +69,7 @@ function App() {
             exact
             component={() => <HomePage LoadingComponent={LoadingComponent} />}
           />
+          {/* delete me */}
           <SecureRoute path="/example-list" component={ExampleListPage} />{' '}
```
Comment (williamschwindt) [inline src/index.js]: I think you can just remove this route now.

**Review by williamschwindt — APPROVED** [review]

**Pull request merged**